app/graphql/types/embed/audio_tag_embed.rb

```diff
@@ -0,0 +1,5 @@
+class Types::Embed::AudioTagEmbed < Types::BaseObject
```

Comment:
Why the `Tag`?

Reply:
seemed kind of strange just being `AudioEmbed` and on the docs it used tag in the description. i.e: `The og:video tag has the identical tags` which I kinda liked. I can rename though.